sql/moz-fx-data-shared-prod/org_mozilla_ios_firefox_derived/legacy_metrics_v1/query.sql

```diff
@@ -0,0 +1,206 @@
+CREATE TEMPORARY FUNCTION labeled_counter(
+  `values` ARRAY<STRUCT<key STRING, value INT64>>,
+  labels ARRAY<STRING>
+) AS (
+  (
+    WITH summed AS (
+      SELECT
+        IF(a.key IN (SELECT * FROM UNNEST(labels)), a.key, "__unknown__") AS k,
+        SUM(a.value) AS v
+      FROM
+        UNNEST(`values`) AS a
```

Comment:
It might make more sense for this to be `FROM UNNEST(labels)`, and then get `SUM(map.get_key(...))` (and we can avoid that subquery)

Reply:
Is it possible to get the `__unknown__` key in this case though?

Reply:
Glean uses `__unknown__` as the default key, so I think this is appropriate. I think it should be a `group by k`, good catch.